### PR TITLE
[docker] Update README to fix links and small improvement

### PR DESCRIPTION
**Pull request by dpgaspar**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [X] Documentation

### SUMMARY
Fixes a couple of links and adds docs to propose a way to monitor the progress of superset initialisation procedure

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by dpgaspar** [comment]
just saw it now, part of this is a bit duplicated from #9511

File: docker/README.md [hunk 1]
```diff
@@ -65,6 +65,10 @@ docker-compose up
 After several minutes for superset initialization to finish, you can open a browser and view [`http://localhost:8088`](http://localhost:8088) 
 to start your journey.
 
+To check the progression of the init procedures run:
+
+`docker logs preset_fork_superset_superset-init_1 -f`
```
Comment (craig-rueda) [inline docker/README.md:70]: Is the container actually called "preset_fork_xxx"?

Comment (dpgaspar) [reply docker/README.md:70]: work on my laptop, lol

Comment (dpgaspar) [reply docker/README.md]: named the containers like on #9547

**Comment by willbarrett** [comment]
Is this still a thing @dpgaspar ? Should we close?

**Pull request closed without merging**